examples/remote_execution/buildbarn/.buckconfig

```diff
@@ -2,13 +2,10 @@
 root = .
 prelude = prelude
 
-[buck2]
-digest_algorithms = SHA256
```

Comment:
Possibly, though I think it's a bit unlikely that we'd make this a priority (we've got no use for it internally, but also that would be a pretty substantial overhaul of how we start up things, since we need this config very early on, which is not true of the RE client).

Reply:
At the moment that's true, there's not much gain in consuming GetCapabilities only to look at which digest function to use when SHA256 is pretty much universally supported.

But looking forward a little, the RE api recently adopted [SHA256TREE](https://github.com/bazelbuild/remote-apis/pull/235) as a digest function which is a precursor to add new functionality to the remote api [ConcatenateBlobs and SplitBlobs](https://github.com/bazelbuild/remote-apis/pull/233). Those will allow partial transfer of larger blobs and ultimately become a more efficient replacement of bytestream.write.

There is of course no hurry, I just thought it might be worth poking the project on a trajectory that meshes well with how the protocol is moving.

Reply:
> But looking forward a little, the RE api recently adopted https://github.com/bazelbuild/remote-apis/pull/235 as a digest function which is a precursor to add new functionality to the remote api https://github.com/bazelbuild/remote-apis/pull/233. Those will allow partial transfer of larger blobs and ultimately become a more efficient replacement of bytestream.write.

Sure, my point there is primarily that actually configuring the hashing scheme you use is rather the trivial bit, there's a lot more to actually implementing describing what you're doing than inferring a configuration :)